lib/solaar/ui/__init__.py

```diff
@@ -88,8 +92,6 @@ def ui_async(function, *args, **kwargs):
 #
 #
 
-from . import notify, tray, window
-
```

Comment:
window uses ui_async so this import can't be moved up in the file

Reply:
@FFY00 To keep the style cops happy, the import needs to be hidden in an `if True:`
I tried to figure out how to refactor, but it was becoming quite a mess and this is much easier for now.

Reply:
I added `# isort:skip` to make isort skip the line and `# noqa: E402` to make flake8 ignore `E402` (`module level import not at top of file`).

Reply:
OK